apps/web-giddh/src/app/new-inventory/component/adjust-inventory/adjust-inventory.component.ts

```diff
@@ -543,6 +551,7 @@ export class AdjustInventoryComponent implements OnInit {
         this.stockReportRequest.count = API_COUNT_LIMIT;
         this.stockReportRequest.inventoryType = this.inventoryType?.toUpperCase();
         this.stockReportRequest.branchUniqueNames = this.generalService.currentBranchUniqueName ? [this.generalService.currentBranchUniqueName] : [];
+        this.stockReportRequest.warehouseUniqueNames = this.adjustInventoryCreateEditForm.get('warehouseUniqueName')?.value ? [this.adjustInventoryCreateEditForm.get('warehouseUniqueName')?.value] : [];
```

Comment:
create const variable for "this.adjustInventoryCreateEditForm.get('warehouseUniqueName')?.value"

and use below line